scripts/sct_maths.py

```diff
@@ -114,16 +114,21 @@ def get_parser():
     thresholding.add_argument(
         "-adap",
         metavar=Metavar.list,
-        help="R|Threshold image using Adaptive algorithm (from skimage). Separate following arguments with ',':"
-             "\n Block size: Odd size of pixel neighborhood which is used to calculate the threshold value (e.g. 3, 7, 21, ...)"
-             "\n Offset: Constant subtracted from weighted mean of neighborhood to calculate the local threshold value. Suggested offset is 0.",
+        type=list_type(',', int),
+        help="R|Threshold image using Adaptive algorithm (from skimage). Provide 2 values separated by ',' that "
+             "correspond to the parameters below. For example, '-adap 7,0' corresponds to a block size of 7 and an "
+             "offset of 0.\n"
+             "  - Block size: Odd size of pixel neighborhood which is used to calculate the threshold value. \n"
+             "  - Offset: Constant subtracted from weighted mean of neighborhood to calculate the local threshold "
+             "value. Suggested offset is 0.",
         required=False)
     thresholding.add_argument(
         "-otsu-median",
+        metavar=Metavar.list,
+        type=list_type(',', int),
         help="R|Threshold image using Median Otsu algorithm. Separate following arguments with ',':"
              "\n Size of the median filter (e.g. 2, 3)"
              "\n Number of iterations (e.g. 3, 4, 5)\n",
```

Comment:
The examples here don't make sense anymore. They should, at least, read "(e.g. 2,3)" or "(e.g. 3, 4, 5)" but would be better written "(e.g. -otsu-median 3,4,5)". I also don't understand how there can be three numbers here when otsu_median() only looks at two.

Reply:
I know this derails a bit from the focus of the PR but we should make sure it makes sense.

Reply:
I very much agree. This is a similar concern to what happened with Julien's comments earlier, i.e. https://github.com/neuropoly/spinalcordtoolbox/pull/2901#discussion_r493109814